Assets/HoloToolkit/Input/Scripts/GazeManager.cs

```diff
@@ -113,24 +113,24 @@ private void UpdateRaycast()
         }
 
         /// <summary>
-        /// Updates the focus point for every frame.
+        /// Adds the stabilization plane modifier if it's enabled and if it doesn't exist yet.
         /// </summary>
         private void UpdateStabilizationPlane()
         {
+            // We want to use the stabilization logic.
             if (SetStabilizationPlane)
             {
-                // Calculate the delta between camera's position and current hit position.
-                float focusPointDistance = (gazeOrigin - Position).magnitude;
-                float lerpPower = focusPointDistance > lastHitDistance
-                    ? LerpStabilizationPlanePowerFarther
-                    : LerpStabilizationPlanePowerCloser;
-
-                // Smoothly move the focus point from previous hit position to new position.
-                lastHitDistance = Mathf.Lerp(lastHitDistance, focusPointDistance, lerpPower * Time.deltaTime);
-
-                Vector3 newFocusPointPosition = gazeOrigin + (gazeDirection * lastHitDistance);
-
-                HolographicSettings.SetFocusPointForFrame(newFocusPointPosition, -gazeDirection);
+                // And it doesn't exist.
+                if (!stabilizationExists)
```

Comment:
This extra state dangerously redundant. Isn't the `StabilizationPlaneModifier.Instance == null` test enough?